src/SonarScanner.MSBuild/Program.cs

```diff
@@ -27,6 +28,7 @@ public static class Program
     private const int ErrorCode = 1;
     private const int SuccessCode = 0;
 
+    [ExcludeFromCodeCoverage]
```

Comment:
Can you put this under test instead? It is important to forward the result from `Execute` as a return code for the exe. You can make `Execute` `virtual` and create a parameterized test where Execute returns a bunch of numbers that need all to be forwarded and returned unchanged from Main.

Reply:
Or simply do not specify any args, or specify invalid ones, which should fail early (presumably)

Reply:
I'm fine with exclusion too